frontend/src/routes/(admin)/oauth/authorize/index.tsx

```diff
@@ -26,50 +26,41 @@ export const clientLoader = loader$<Promise<Client>, { DATABASE: D1Database }>(a
 	return client
 })
 
-export const userLoader = loader$<
-	Promise<{ email: string; avatar: URL; name: string; url: URL }>,
-	{ DATABASE: D1Database; domain: string }
->(async ({ cookie, platform, html, request, redirect, text }) => {
-	const jwt = cookie.get('CF_Authorization')
-	if (jwt === null) {
-		throw html(500, getErrorHtml('Missing Authorization'))
-	}
-	let payload: access.JWTPayload
-	try {
-		// TODO: eventually, verify the JWT with Access, however this
-		// is not critical.
-		payload = access.getPayload(jwt.value)
-	} catch (e: unknown) {
-		const error = e as { stack: string; cause: string }
-		console.warn(error.stack, error.cause)
-		throw html(500, getErrorHtml('Failed to validate Access JWT'))
-	}
-
-	if (!payload.email) {
-		throw html(500, getErrorHtml("The Access JWT doesn't contain an email"))
-	}
+export const userLoader = loader$<Promise<{ email: string; avatar: URL; name: string; url: URL }>>(
+	async ({ cookie, platform, html, request, redirect, text }) => {
+		const jwt = cookie.get('CF_Authorization')
```

Comment:
Where does this get set as a cookie? AFAIK this is always passed around as a header, just wondering if some previous code already exists that cookies the value on request

Reply:
I __think__ that this is just qwikCity making it more convenient for devs to access the requests cookie (moving them from the headers to this field and also parsing them)

isn't that the case?